AdminWebsite/AdminWebsite/ClientApp/src/app/error/error.component.html

```diff
@@ -1,6 +1,31 @@
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
-    <p class="govuk-body">Please try again later.</p>
-  </div>
+  <ng-container *ngIf="hasConnection$ | async">
```

Comment:
Could `*ngIf="hasConnection$ | async;noConnection"` work here?

Reply:
Using a ngTemplate? Probably, yeah - you might need to use parens, though, like this `*ngIf="(hasConnection$ | async); else noConnection"`